### PR TITLE
Jira docs 3.0

### PR DESCRIPTION
**Pull request by justinegeffen**
Removed extraneous page and fixed inconsistent TOC capitalization.

### CODE REVIEW

**Review by jfrerich — APPROVED** [review]
LGTM

**Review by mickmister — APPROVED** [review]

File: docs/setup/configuration.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ To control Mattermost channel subscriptions, use the command `/jira subscribe` i
 1. As a Jira System Administrator, go to **Jira Settings > System > WebHooks**.
```
Comment (bbodenmiller) [inline docs/setup/configuration.md:21]: Would be helpful if step 2 mentioned they could run `/jira webhook jiraURL` to get setup instructions.

File: docs/setup/configuration.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ To control Mattermost channel subscriptions, use the command `/jira subscribe` i
...
    * For older versions of Jira, click the gear icon in bottom left corner, then go to **Advanced > WebHooks**.
 2. Click **Create a WebHook** to create a new webhook. 
-3. Enter a **Name** for the webhook and add the Jira webhook URL [https://SITEURL/plugins/jira/api/v2/webhook?secret=WEBHOOKSECRET](https://SITEURL/plugins/jira/api/v2/webhook?secret=WEBHOOKSECRET) as the **URL**.
+3. Enter a **Name** for the webhook and add the Jira plugin webhook URL [https://SITEURL/plugins/jira/api/v2/webhook?secret=WEBHOOKSECRET](https://SITEURL/plugins/jira/api/v2/webhook?secret=WEBHOOKSECRET) as the **URL**.
```
Comment (bbodenmiller) [inline docs/setup/configuration.md:24]: Doesn't give example of new URL format

**Comment by mickmister** [comment]
@justinegeffen Sorry when I merged https://github.com/mattermost/mattermost-plugin-jira/pull/766 it caused conflicts here

**Comment by mickmister** [comment]
@justinegeffen Is there anything blocking this for merging?

**Comment by justinegeffen** [comment]
> @justinegeffen Is there anything blocking this for merging?

Nope, I think we're good to go! :)

**Pull request merged**